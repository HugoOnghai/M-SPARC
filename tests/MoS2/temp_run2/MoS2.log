
                            < M A T L A B (R) >
                  Copyright 1984-2019 The MathWorks, Inc.
                  R2019a (9.6.0.1072779) 64-bit (glnxa64)
                               March 8, 2019

 
To get started, type doc.
For product information, visit www.mathworks.com.
 

 Reading .inpt file ...


<INPT>
# $ cat MoS2.inpt
# nprocs: 48
# Test: MoS2 #
LATVEC:
0 0 1
1 0 0
-0.5 0.8666 0

CELL: 30 6.1 6.1 
MESH_SPACING: 0.20
FD_ORDER: 12
BC: D P P
KPOINT_GRID: 1 3 3
EXCHANGE_CORRELATION: GGA_PBE
ELEC_TEMP_TYPE: fermi-dirac
ELEC_TEMP: 315.773
TOL_SCF: 1e-6
#ELEC_TEMP: 
CALC_STRESS: 1
PRINT_FORCES: 1
PRINT_ATOMS: 0
MIXING_VARIABLE: potential
MIXING_PRECOND: none

# MD
#MD_FLAG:  1
#MD_METHOD: NVT_NH
#MD_TIMESTEP: 1
#MD_NSTEP: 10
#PRINT_RESTART_FQ: 3
#RESTART_FLAG: 0

<\INPT>

 Reading .ion file ...
 Number of atom types : 2
 Total number of atoms: 3
 Default atomic mass for Mo is 95.950000
 Default atomic mass for S is 32.067500
 pseudo_dir # 1 = Mo.psp8
 pseudo_dir # 2 = S.psp8

 COORD:
   0.070149695000000   3.296864329500000  15.000000000000000
   2.806003050000000   1.762084904600000  18.655220000000000
   3.050003050000000   1.762084904600000  12.344780000000000

   5.403031739800300

   0.031202687423158

 kpoint grid before symmetry:
                   0  -0.343343459408721  -0.343343459408721
                   0                   0  -0.343343459408721
                   0   0.343343459408721  -0.343343459408721
                   0  -0.343343459408721                   0
                   0                   0                   0
                   0   0.343343459408721                   0
                   0  -0.343343459408721   0.343343459408721
                   0                   0   0.343343459408721
                   0   0.343343459408721   0.343343459408721

 kpoint grid after symmetry:
                   0                   0                   0
                   0   0.343343459408721                   0
                   0  -0.343343459408721   0.343343459408721
                   0                   0   0.343343459408721
                   0   0.343343459408721   0.343343459408721

## Chebyshev polynomial degree not provided, finding npl ...
## Based on the mesh size, npl is set to: 35
## Number of states not provided, finding Nev ...
## Based on the number of electrons, Nev is set to: 21
## Poisson tolerance not provided, choosing poisson_tol ...
## poisson_tol is set to: 1.000e-08
## Pseudocharge tolerance not provided, choosing pseudocharge_tol ...
## pseudocharge_tol is set to: 1.000e-08
 Creating differentiation matrices ...
 Done. (7.729 sec)

 Estimated memory usage:
 Total:    1.22 GB
 orbitals            :  870.94 MB
 sparse matrices     :  350.45 MB
 global-size vectors :   14.52 MB
 mixing histories    :   14.52 MB

 Finding rb for Mo ...
 rb = {2.800 2.843 2.843}, int_b = -14.001092001543226, err_rb = 1.092e-03
 rb = {3.000 3.043 3.043}, int_b = -14.000106116110388, err_rb = 1.061e-04
 rb = {3.200 3.243 3.243}, int_b = -14.000131249719349, err_rb = 1.312e-04
 rb = {3.400 3.443 3.443}, int_b = -14.000095516061766, err_rb = 9.552e-05
 rb = {3.600 3.643 3.643}, int_b = -14.000083817557449, err_rb = 8.382e-05
 rb = {3.800 3.843 3.843}, int_b = -14.000069869440848, err_rb = 6.987e-05
 rb = {4.000 4.043 4.043}, int_b = -14.000059491302972, err_rb = 5.949e-05
 rb = {4.200 4.243 4.243}, int_b = -14.000052154526056, err_rb = 5.215e-05
 rb = {4.400 4.443 4.443}, int_b = -14.000047442620859, err_rb = 4.744e-05
 rb = {4.600 4.643 4.643}, int_b = -14.000043593232071, err_rb = 4.359e-05
 rb = {4.800 4.843 4.843}, int_b = -14.000041396357968, err_rb = 4.140e-05
 rb = {5.000 5.043 5.043}, int_b = -14.000036833091084, err_rb = 3.683e-05
 rb = {5.200 5.243 5.243}, int_b = -14.000032829272236, err_rb = 3.283e-05
 rb = {5.400 5.443 5.443}, int_b = -14.000029074686337, err_rb = 2.907e-05
 rb = {5.600 5.643 5.643}, int_b = -14.000028177802678, err_rb = 2.818e-05
 rb = {5.800 5.843 5.843}, int_b = -14.000024139731803, err_rb = 2.414e-05
 rb = {6.000 6.043 6.043}, int_b = -14.000018233490396, err_rb = 1.823e-05
 rb = {6.200 6.243 6.243}, int_b = -14.000016169113195, err_rb = 1.617e-05
 rb = {6.400 6.443 6.443}, int_b = -14.000014665721306, err_rb = 1.467e-05
 rb = {6.600 6.643 6.643}, int_b = -14.000013146255307, err_rb = 1.315e-05
 rb = {6.800 6.843 6.843}, int_b = -14.000014001172568, err_rb = 1.400e-05
 rb = {7.000 7.043 7.043}, int_b = -13.999998243881695, err_rb = 1.756e-06
 rb = {7.200 7.243 7.243}, int_b = -14.000000370321080, err_rb = 3.703e-07
 rb = {7.400 7.443 7.443}, int_b = -13.999999927834251, err_rb = 7.217e-08
 rb = {7.600 7.643 7.643}, int_b = -14.000000009946367, err_rb = 9.946e-09
 rb = {7.600 7.643 7.643}
 Finding rb for S ...
 rb = {2.400 2.437 2.437}, int_b = -5.999383434374064, err_rb = 6.166e-04
 rb = {2.600 2.637 2.637}, int_b = -5.999772222124865, err_rb = 2.278e-04
 rb = {2.800 2.837 2.837}, int_b = -5.999946952436909, err_rb = 5.305e-05
 rb = {3.000 3.037 3.037}, int_b = -6.000008812291933, err_rb = 8.812e-06
 rb = {3.200 3.237 3.237}, int_b = -6.000028706824276, err_rb = 2.871e-05
 rb = {3.400 3.437 3.437}, int_b = -6.000031642032397, err_rb = 3.164e-05
 rb = {3.600 3.637 3.637}, int_b = -6.000029569051675, err_rb = 2.957e-05
 rb = {3.800 3.837 3.837}, int_b = -6.000025373944016, err_rb = 2.537e-05
 rb = {4.000 4.037 4.037}, int_b = -6.000021072252862, err_rb = 2.107e-05
 rb = {4.200 4.237 4.237}, int_b = -6.000018028382754, err_rb = 1.803e-05
 rb = {4.400 4.437 4.437}, int_b = -6.000016480752667, err_rb = 1.648e-05
 rb = {4.600 4.637 4.637}, int_b = -6.000015507209615, err_rb = 1.551e-05
 rb = {4.800 4.837 4.837}, int_b = -6.000015677560460, err_rb = 1.568e-05
 rb = {5.000 5.037 5.037}, int_b = -6.000014106067415, err_rb = 1.411e-05
 rb = {5.200 5.237 5.237}, int_b = -6.000013008440810, err_rb = 1.301e-05
 rb = {5.400 5.437 5.437}, int_b = -6.000012139650035, err_rb = 1.214e-05
 rb = {5.600 5.637 5.637}, int_b = -6.000013362995111, err_rb = 1.336e-05
 rb = {5.800 5.837 5.837}, int_b = -6.000012190326671, err_rb = 1.219e-05
 rb = {6.000 6.037 6.037}, int_b = -6.000009339941093, err_rb = 9.340e-06
 rb = {6.200 6.237 6.237}, int_b = -6.000009090619262, err_rb = 9.091e-06
 rb = {6.400 6.437 6.437}, int_b = -6.000009063815738, err_rb = 9.064e-06
 rb = {6.600 6.637 6.637}, int_b = -6.000008802885319, err_rb = 8.803e-06
 rb = {6.800 6.837 6.837}, int_b = -6.000010104707603, err_rb = 1.010e-05
 rb = {7.000 7.037 7.037}, int_b = -5.999998728444395, err_rb = 1.272e-06
 rb = {7.200 7.237 7.237}, int_b = -6.000000268218048, err_rb = 2.682e-07
 rb = {7.400 7.437 7.437}, int_b = -5.999999947878257, err_rb = 5.212e-08
 rb = {7.600 7.637 7.637}, int_b = -6.000000007159445, err_rb = 7.159e-09
 rb = {7.600 7.637 7.637}

 ###############################################################
 Relaxation step number: 1 

 Starting pseudocharge generation and self energy calculation...
 Integration b = 26.000000007541

 Integration b_ref = 25.999999999956

 ****************************************
 *          Eself_ref = 478.738373       *
 ****************************************
 Done. (3.621190 s)
 Time for b calculation: 3.637 seconds.

 Starting calculating nonlocal projectors ...
 Done. (2.044779 s)

 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ 
 Starting SCF iteration...
 AAR converged to a relative residual of 8.61198e-09 in 277 iterations.
 Poisson problem took 30.395114s
 ============================================= 
 Relaxation iteration:  1
 SCF iteration number:  1, Chebyshev cycle: 1 
 ============================================= 
 Fermi energy = 4.566008
 This SCF iteration took 210.441 s.

 ============================================= 
 Relaxation iteration:  1
 SCF iteration number:  1, Chebyshev cycle: 2 
 ============================================= 
 Fermi energy = 0.340502
 This SCF iteration took 175.047 s.

 ============================================= 
 Relaxation iteration:  1
 SCF iteration number:  1, Chebyshev cycle: 3 
 ============================================= 
 Fermi energy = -0.165093
 This SCF iteration took 174.356 s.

 ============================================= 
 Relaxation iteration:  1
 SCF iteration number:  1, Chebyshev cycle: 4 
 ============================================= 
 Fermi energy = -0.203120
 AAR converged to a relative residual of 9.51875e-09 in 277 iterations.
 Poisson problem took 30.154885s
 Escc = 0.99384886
 Etot = -88.99990690
 Eatom = -29.66663563
 Error in SCF iteration: 1.7800e-01 
 This SCF iteration took 205.494 s.

 ========================= 
 Relaxation iteration:  1 
 SCF iteration number:  2 
 ========================= 
 Fermi energy = -0.196079
 AAR converged to a relative residual of 9.35387e-09 in 295 iterations.
 Poisson problem took 32.216629s
 Escc = -1.44857219
 Etot = -88.83084937
 Eatom = -29.61028312
 Error in SCF iteration: 5.2945e-01 
 This SCF iteration took 208.101 s.

 ========================= 
 Relaxation iteration:  1 
 SCF iteration number:  3 
 ========================= 
 Fermi energy = -0.203308
 AAR converged to a relative residual of 9.35241e-09 in 307 iterations.
 Poisson problem took 33.909138s
 Escc = 1.51216473
 Etot = -89.08015522
 Eatom = -29.69338507
 Error in SCF iteration: 5.5191e-02 
 This SCF iteration took 207.937 s.

 ========================= 
 Relaxation iteration:  1 
 SCF iteration number:  4 
 ========================= 
 Fermi energy = -0.195338
 AAR converged to a relative residual of 5.43604e-09 in 241 iterations.
 Poisson problem took 26.009401s
 Escc = 0.33197358
 Etot = -89.08191389
 Eatom = -29.69397130
 Error in SCF iteration: 4.7813e-02 
 This SCF iteration took 200.491 s.

 ========================= 
 Relaxation iteration:  1 
 SCF iteration number:  5 
 ========================= 
 Fermi energy = -0.192460
 AAR converged to a relative residual of 9.94444e-09 in 239 iterations.
 Poisson problem took 25.858569s
 Escc = 1.11815950
 Etot = -89.08143095
 Eatom = -29.69381032
 Error in SCF iteration: 3.9995e-02 
 This SCF iteration took 200.778 s.

 ========================= 
 Relaxation iteration:  1 
 SCF iteration number:  6 
 ========================= 
 Fermi energy = -0.183612
 AAR converged to a relative residual of 9.14095e-09 in 247 iterations.
 Poisson problem took 26.661888s
 Escc = -0.34430809
 Etot = -89.08358069
 Eatom = -29.69452690
 Error in SCF iteration: 1.5774e-02 
 This SCF iteration took 201.335 s.

 ========================= 
 Relaxation iteration:  1 
 SCF iteration number:  7 
 ========================= 
 Fermi energy = -0.183766
 AAR converged to a relative residual of 6.43899e-09 in 247 iterations.
 Poisson problem took 26.562906s
 Escc = -0.02772463
 Etot = -89.08394978
 Eatom = -29.69464993
 Error in SCF iteration: 5.1754e-03 
 This SCF iteration took 200.730 s.

 ========================= 
 Relaxation iteration:  1 
 SCF iteration number:  8 
 ========================= 
 Fermi energy = -0.184186
 AAR converged to a relative residual of 8.94746e-09 in 199 iterations.
 Poisson problem took 21.496921s
 Escc = 0.00307509
 Etot = -89.08396799
 Eatom = -29.69465600
 Error in SCF iteration: 8.4820e-04 
 This SCF iteration took 196.044 s.

 ========================= 
 Relaxation iteration:  1 
 SCF iteration number:  9 
 ========================= 
 Fermi energy = -0.184244
 AAR converged to a relative residual of 8.19229e-09 in 169 iterations.
 Poisson problem took 18.113740s
 Escc = -0.00343420
 Etot = -89.08396922
 Eatom = -29.69465641
 Error in SCF iteration: 1.4913e-04 
 This SCF iteration took 191.286 s.

 ========================= 
 Relaxation iteration:  1 
 SCF iteration number: 10 
 ========================= 
 Fermi energy = -0.184380
 AAR converged to a relative residual of 9.72105e-09 in 133 iterations.
 Poisson problem took 14.286825s
 Escc = 0.00023546
 Etot = -89.08396881
 Eatom = -29.69465627
 Error in SCF iteration: 1.9117e-04 
 This SCF iteration took 198.592 s.

 ========================= 
 Relaxation iteration:  1 
 SCF iteration number: 11 
 ========================= 
 Fermi energy = -0.184380
 AAR converged to a relative residual of 9.54833e-09 in 97 iterations.
 Poisson problem took 10.488891s
 Escc = 0.00048055
 Etot = -89.08396898
 Eatom = -29.69465633
 Error in SCF iteration: 3.9401e-05 
 This SCF iteration took 192.723 s.

 ========================= 
 Relaxation iteration:  1 
 SCF iteration number: 12 
 ========================= 
 Fermi energy = -0.184376
 AAR converged to a relative residual of 9.46898e-09 in 97 iterations.
 Poisson problem took 10.522732s
 Escc = -0.00000046
 Etot = -89.08396963
 Eatom = -29.69465654
 Error in SCF iteration: 6.4193e-06 
 This SCF iteration took 192.381 s.

 ========================= 
 Relaxation iteration:  1 
 SCF iteration number: 13 
 ========================= 
 Fermi energy = -0.184373
 AAR converged to a relative residual of 7.1385e-09 in 49 iterations.
 Poisson problem took 5.341783s
 Escc = 0.00002308
 Etot = -89.08396921
 Eatom = -29.69465640
 Error in SCF iteration: 3.0534e-06 
 This SCF iteration took 185.885 s.

 ========================= 
 Relaxation iteration:  1 
 SCF iteration number: 14 
 ========================= 
 Fermi energy = -0.184375
 AAR converged to a relative residual of 9.01274e-09 in 25 iterations.
 Poisson problem took 2.764802s
 Escc = 0.00005287
 Etot = -89.08396958
 Eatom = -29.69465653
 Error in SCF iteration: 1.8076e-06 
 This SCF iteration took 185.337 s.

 ========================= 
 Relaxation iteration:  1 
 SCF iteration number: 15 
 ========================= 
 Fermi energy = -0.184374
 AAR converged to a relative residual of 7.94165e-09 in 19 iterations.
 Poisson problem took 2.114339s
 Escc = 0.00001954
 Etot = -89.08396942
 Eatom = -29.69465647
 Error in SCF iteration: 6.9012e-07 
 This SCF iteration took 184.666 s.


 Finished SCF iteration in 15 steps!
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ 

 **********************************************************
 *          Energy per unit cell = -89.083969422 Ha.       *
 *          Energy per atom = -29.694656474 Ha.            *
 **********************************************************

 Starting atomic force calculation ... 
 local force calculation: 3.375 s
 ***********************************************************
 *                      Atomic Force                       *
 ***********************************************************
 Drift free forces (Ha/Bohr):
  -0.023507932604067   0.038942929647769   0.107268309186287
   0.012522958636771  -0.011088799667922  -0.068722682545575
   0.010984973967296  -0.027854129979847  -0.038545626640712

 Max magnitude of forces (Ha/Bohr):   0.116514654960452


 Time for calculating forces: 9.309142 s.

["Stress in Ha/Bohr**2"


]   0.001420698301238  -0.001037360751317
  -0.001037360751317   0.004626459325708


["Stress in GPa (equivalent to all periodic system)"


]   1.393279567357237  -1.017340231581666
  -1.017340231581666   4.537171081362869


 Time for calculating stress: 28.541025 s.

 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ 
 Final atomic positions (Cartesian) are as follows:
 15.000000 	  1.972333 	  3.806260 
 18.655220 	  3.822669 	  2.034343 
 12.344780 	  4.066669 	  2.034343 

 Run-time of the program: 3598.537290 seconds

ans = 

  struct with fields:

                  cell_typ: 2
                   lat_vec: [3x3 double]
                  metric_T: [3x3 double]
                    grad_T: [3x3 double]
                    lapc_T: [3x3 double]
                      Jacb: 0.866168945680640
                        L1: 30
                        L2: 6.100000000000000
                        L3: 6.100000000000000
                        Nx: 151
                        Ny: 30
                        Nz: 30
                         N: 135900
                      ecut: -1
              mesh_spacing: 0.200000000000000
                   kptgrid: [5x3 double]
                  kptshift: [0 0 0]
                      nkpt: [1 3 3]
                     tnkpt: 5
                      wkpt: [5x1 double]
                        BC: 3
                       BCx: 1
                       BCy: 0
                       BCz: 0
                      isBS: 0
                   lattice: 'undefined'
                   SCF_tol: 1.000000000000000e-06
                       Nev: 21
               poisson_tol: 1.000000000000000e-08
          pseudocharge_tol: 1.000000000000000e-08
                       Cst: 27.211386245987999
              Ha_Bohr3_GPa: 2.942101569650548e+04
                        kB: 3.166811563455600e-06
               elec_T_type: 0
                      Temp: 3.157730000000000e+02
                       bet: 1.000006412214051e+03
                       npl: 35
                       FDn: 6
                    rc_ref: 0.500000000000000
              max_relax_it: 100
                dbg_switch: 0
                        xc: 2
                 Nelectron: 26
                     n_typ: 2
                TimeRevSym: 1
                  spin_typ: 0
                     nspin: 1
              CheFSI_Optmz: 0
          chefsibound_flag: 0
               FixRandSeed: 0
                rhoTrigger: 4
                 NetCharge: 0
                 MAXIT_SCF: 100
                 MINIT_SCF: 3
             MAXIT_POISSON: 1000
            accuracy_level: -1
     target_force_accuracy: -1
    target_energy_accuracy: -1
                 TOL_RELAX: 5.000000000000000e-04
               TOL_LANCZOS: 0.010000000000000
               precond_tol: 4.088636571742514e-05
               precond_kTF: 1
            precond_thresh: 0.250000000000000
            precond_fitpow: 2
          precond_resta_q0: 1.360000000000000
          precond_resta_Rs: 2.760000000000000
            MixingVariable: 1
             MixingPrecond: 0
                  Pf_guess: []
             MixingHistory: 7
           MixingParameter: 0.300000000000000
     MixingParameterSimple: 0.300000000000000
            PulayFrequency: 1
          PulayRestartFreq: 0
                    TWtime: 1.000000000000000e+09
                 RelaxFlag: 0
                 RelaxMeth: 'LBFGS'
            max_dilatation: 1.200000000000000
            TOL_RELAX_CELL: 5.000000000000000e-06
                    MDFlag: 0
               RestartFlag: 0
                    MDMeth: 'NVE'
                     MD_dt: 1
                  MD_Nstep: 0
                     ion_T: -1
                thermos_TF: -1
              ion_elec_eqT: 1
              ion_vel_dstr: 2
                NLCG_sigma: 0.500000000000000
                     qmass: 1
                 L_history: 20
               L_finit_stp: 0.005000000000000
                  L_maxmov: 0.200000000000000
               L_autoscale: 1
                 L_lineopt: 1
                   L_icurv: 1
                   FIRE_dt: 1
                 FIRE_mass: 1
               FIRE_maxmov: 0.200000000000000
               Calc_stress: 1
                 Calc_pres: 0
            PrintForceFlag: 1
          PrintAtomPosFlag: 0
          PrintAtomVelFlag: 1
         PrintElecDensFlag: 0
            PrintEigenFlag: 0
                PrintMDout: 1
             PrintRelaxout: 1
              Printrestart: 1
           Printrestart_fq: 1
                       xin: 0
                       yin: 0
                       zin: 0
                     twist: 0
                  filename: 'MoS2'
            inputfile_path: '.'
                        XC: 'GGA_PBE'
                       Atm: [1x2 struct]
                     n_atm: 3
                     Atoms: [3x3 double]
           mvAtmConstraint: [3x3 double]
                    IsFrac: [2x1 double]
                    IsSpin: [2x1 double]
            is_default_psd: 0
                  temp_tol: 1.000000000000000e-12
                 xc_rhotol: 1.000000000000000e-14
                Atoms_init: [3x3 double]
                  lat_uvec: [3x3 double]
                        dx: 0.200000000000000
                        dy: 0.203333333333333
                        dz: 0.203333333333333
                        dV: 0.007162254770839
                        w2: [1x7 double]
                        w1: [1x7 double]
                         W: [135900x1 double]
                    occfac: 2
                Relax_iter: 1
              elecgs_Count: 1
               StressCount: 1
                    amu2au: 1.822888485332371e+03
                    fs2atu: 41.341373336493000
                      I_11: [1921x1 double]
                     II_11: [1921x1 double]
                      V_11: [1921x1 double]
                 isOutl_22: [390x1 logical]
                 isOutr_22: [390x1 logical]
                      I_22: [390x1 double]
                     II_22: [390x1 double]
                      V_22: [390x1 double]
                 isOutl_33: [390x1 logical]
                 isOutr_33: [390x1 logical]
                      I_33: [390x1 double]
                     II_33: [390x1 double]
                      V_33: [390x1 double]
                       I_1: [1770x1 double]
                      II_1: [1770x1 double]
                       V_1: [1770x1 double]
                  isOutl_2: [360x1 logical]
                  isOutr_2: [360x1 logical]
                       I_2: [360x1 double]
                      II_2: [360x1 double]
                       V_2: [360x1 double]
                  isOutl_3: [360x1 logical]
                  isOutr_3: [360x1 logical]
                       I_3: [360x1 double]
                      II_3: [360x1 double]
                       V_3: [360x1 double]
                     G_J_1: [1593000x1 double]
                     G_I_1: [1593000x1 double]
                     G_V_1: [1593000x1 double]
                 G_JOutl_2: [1630800x1 logical]
                 G_JOutr_2: [1630800x1 logical]
                     G_J_2: [1630800x1 double]
                     G_I_2: [1630800x1 double]
                     G_V_2: [1630800x1 double]
                 G_JOutl_3: [1630800x1 logical]
                 G_JOutr_3: [1630800x1 logical]
                     G_J_3: [1630800x1 double]
                     G_I_3: [1630800x1 double]
                     G_V_3: [1630800x1 double]
                   Lap_std: [135900x135900 double]
                    grad_1: [135900x135900 double]
                    grad_2: [135900x135900 double]
                    grad_3: [135900x135900 double]
                LapPreconL: [135900x135900 double]
                LapPreconU: [135900x135900 double]
              memory_usage: 1.311163208000000e+09
               staticfname: 'MoS2.static'
                 suffixNum: 0
                  outfname: 'MoS2.out'
                  parallel: 0
                         b: [135900x1 double]
                     b_ref: [135900x1 double]
                    rho_at: [135900x1 double]
                     Eself: 2.119678144426026e+02
                 Eself_ref: 4.787383725487986e+02
                       V_c: [135900x1 double]
                 PosCharge: 26.000000007541185
                 NegCharge: -26.000000007541185
                    E_corr: -2.507384708110294e-04
                       rho: [135900x1 double]
                      Atom: [1x3 struct]
                       phi: [135900x1 double]
                      e_xc: [135900x1 double]
                 dvxcdgrho: [135900x1 double]
                       Vxc: [135900x1 double]
                      Veff: [135900x1 double]
                         X: [135900x7 double]
                         F: [135900x7 double]
          mixing_hist_fkm1: [135900x1 double]
          mixing_hist_xkm1: [135900x1 double]
                       psi: [135900x21x5 double]
    upper_bound_guess_vecs: [135900x5 double]
                    EigVal: [21x5 double]
                  lambda_f: -0.184373720277624
                       occ: [21x5 double]
                    Etotal: -89.083969422338896
                     Eband: -21.030617565559464
                       Exc: -12.270350995021069
                    Exc_dc: -15.980813420819285
                  Eelec_dc: 1.402042313617173e+02
                      Eent: -2.052306266443638e-10
                      Escc: 1.953698359764207e-05
                   S_Debug: [1x1 struct]
                     force: [3x3 double]
                 abs_force: [3x1 double]
                    Stress: [2x2 double]
             stress_dimred: [2x3 double]

