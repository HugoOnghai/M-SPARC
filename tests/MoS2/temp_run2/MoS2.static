Atomic forces (Ha/Bohr):
     -0.0235079326       0.0389429296       0.1072683092
      0.0125229586      -0.0110887997      -0.0687226825
      0.0109849740      -0.0278541300      -0.0385456266
Stress (Ha/Bohr**2)                :
  1.4206983012E-03  -1.0373607513E-03 
 -1.0373607513E-03   4.6264593257E-03 
Stress equiv. all periodic (GPa)   :
  1.3932795674E+00  -1.0173402316E+00 
 -1.0173402316E+00   4.5371710814E+00 
