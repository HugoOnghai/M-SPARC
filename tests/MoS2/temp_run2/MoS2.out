***************************************************************************
*                          M-SPARC (Feb 20, 2020)                         *
*   Copyright (c) 2019 Material Physics & Mechanics Group, Georgia Tech   *
*           Distributed under GNU General Public License 3 (GPL)          *
*                Date: 11-May-2021  Start time: 21:08:51                  *
***************************************************************************
                           Input parameters                                
***************************************************************************
CELL: 30.000000 6.100000 6.100000 
LATVEC:
0.000000000000000 0.000000000000000 1.000000000000000 
1.000000000000000 0.000000000000000 0.000000000000000 
-0.500000000000000 0.866600000000000 0.000000000000000 
FD_GRID: 150 30 30
FD_ORDER: 12
BC: D P P
KPOINT_GRID: 1 3 3
KPOINT_SHIFT: 0 0 0
ELEC_TEMP_TYPE: fermi-dirac
SMEARING: 0.000999994
CHEB_DEGREE: 35
NSTATES: 21
EXCHANGE_CORRELATION: GGA_PBE
CALC_STRESS: 1
MAXIT_SCF: 100
TOL_SCF: 1.00E-06
TOL_POISSON: 1.00E-08
TOL_LANCZOS: 1.00E-02
TOL_PSEUDOCHARGE: 1.00E-08
MIXING_VARIABLE: potential
MIXING_PRECOND: none
MIXING_PARAMETER: 0.30
MIXING_HISTORY: 7
PULAY_FREQUENCY: 1
PULAY_RESTART_FREQ: 0
REFERENCE_CUTOFF: 0.50
RHO_TRIGGER: 4
FIX_RAND: 0
PRINT_FORCES: 1
PRINT_ATOMS: 0
PRINT_EIGEN: 0
PRINT_DENSITY: 0
OUTPUT_FILE: MoS2.out
***************************************************************************
                             Initialization                                
***************************************************************************
Mesh spacing in x-direction        :  0.200000 (Bohr)
Mesh spacing in y-direction        :  0.203333 (Bohr)
Mesh spacing in z direction        :  0.203333 (Bohr)
Number of symmetry adapted k-points:  5
Output printed to                  :  MoS2.out
Total number of atom types         :  2
Total number of atoms              :  3
Total number of electrons          :  26
Atom type 1  (valence electrons)   :  Mo 14
Pseudopotential                    :  Mo.psp8
lloc                               :  4
Pseudocharge radii of atom type 1  :  7.60 7.64 7.64
Number of atoms of type 1          :  1
Atom type 2  (valence electrons)   :  S 6
Pseudopotential                    :  S.psp8
lloc                               :  4
Pseudocharge radii of atom type 2  :  7.60 7.64 7.64
Number of atoms of type 2          :  2
Estimated total memory usage       :  1.22 GB
=====================================================================
                    Self Consistent Field (SCF#1)                     
=====================================================================
Iteration     Free Energy (Ha/atom)   SCF Error        Timing (sec)
1            -2.9666635632E+01        1.780E-01        765.336
2            -2.9610283123E+01        5.295E-01        208.098
3            -2.9693385073E+01        5.519E-02        207.935
4            -2.9693971296E+01        4.781E-02        200.489
5            -2.9693810318E+01        3.999E-02        200.774
6            -2.9694526898E+01        1.577E-02        201.333
7            -2.9694649927E+01        5.175E-03        200.728
8            -2.9694655996E+01        8.482E-04        196.042
9            -2.9694656405E+01        1.491E-04        191.284
10           -2.9694656270E+01        1.912E-04        198.590
11           -2.9694656327E+01        3.940E-05        192.721
12           -2.9694656545E+01        6.419E-06        192.378
13           -2.9694656402E+01        3.053E-06        185.883
14           -2.9694656527E+01        1.808E-06        185.334
15           -2.9694656474E+01        6.901E-07        184.664
Total number of SCF: 15    
====================================================================
                                Energy                              
====================================================================
Free energy per atom               : -2.9694656474E+01 (Ha/atom)
Total free energy                  : -8.9083969422E+01 (Ha)
Band structure energy              : -2.1030617566E+01 (Ha)
Exchange correlation energy        : -1.2270350995E+01 (Ha)
Self and correction energy         : -2.1196806518E+02 (Ha)
Entropy*kb*T                       : -2.0523062664E-10 (Ha)
Fermi level                        : -1.8437372028E-01 (Ha)
Average force                      :  7.8684116509E-02 (Ha/Bohr)
Maximum force                      :  1.1651465496E-01 (Ha/Bohr)
Time for force calculation         :  9.309 (sec)
Max stress                         :  4.6264593257E-03 (Ha/Bohr**2)
Max stress equiv. all periodic     :  4.5371710814E+00 (GPa)
Time for stress calculation        :  28.544 (sec)
***************************************************************************
                               Timing info                                 
***************************************************************************
Total walltime                     :  3598.537 sec
___________________________________________________________________________

***************************************************************************
*             Material Physics & Mechanics Group, Georgia Tech            *
*                       PI: Phanish Suryanarayana                         *
*                Main Developers: Qimen Xu, Abhiraj Sharma                *
*     Acknowledgements: U.S. DOE (DE-SC0019410); NSF (1333500,1553212)    *
***************************************************************************
                                                                           
